iot/api-client/http_example/cloudiot_http_example.py

```diff
@@ -83,9 +88,16 @@ def publish_message(
     resp = requests.post(
             publish_url, data=json.dumps(body), headers=headers)
 
+    if (resp.status_code != 200):
```

Comment:
If you wanted, you could use `resp.raise_for_status()`, but this is fine.

Reply:
ack.